### PR TITLE
25 admin merchant show

### LINKED ISSUE #25: Admin Merchant Show
Admin Merchant Show

As an admin,
When I click on the name of a merchant from the admin merchants index page,
Then I am taken to that merchant's admin show page (/admin/merchants/merchant_id)
And I see the name of that merchant

┆Issue is synchronized with this [Trello card](https://trello.com/c/gOTloaax) by [Unito](https://www.unito.io)

### PR DESCRIPTION
**Pull request by SandyyMarie**
Completed User 25 with passing features and tests.
_As an admin,
When I click on the name of a merchant from the admin merchants index page,
Then I am taken to that merchant's admin show page (/admin/merchants/merchant_id)
And I see the name of that merchant_

### CODE REVIEW

**Review by aleish-m — APPROVED** [review]
This looks great, Sandy!! Thanks for knocking this one out while ken and I figured out the kinks on the merchant page for invoices.

File: app/views/admin/merchants/index.html.erb
Comment (aleish-m) [inline app/views/admin/merchants/index.html.erb:3]: I think Brian said we don't need to include `.id` when we use ruby paths I think we can just call `admin_merchant_path(merchant)`, and ruby knows what to pull from that object. But we can play around with that on the next story if we want or you can try it before we pull since there are merge conflicts on the PR ahead of this one.

**Comment by aleish-m** [comment]
I'll merge this once the other merge request is merged and conflicts resolved so we don't introduce any other potential conflicts.

**Review by Penitent0 — APPROVED** [review]
Look great!

**Pull request merged**
